.github/pull_request_template.md

```diff
@@ -1,11 +1,14 @@
-### Related Issue or purpose 
-- issue goes here
+### Proposed changes
+-
+-
 
-### Proposed Changes
-- change 1 
-- change 2 
+<!-- (uncomment if applicable)
```

Comment:
Not sure if it's cool to comment out the headings. And if it would be, not sure if it's cool to comment out multiple at once.

Reply:
I agree, The Idea here is to reduce the friction for our (internal) PRs. Also, e.g., a small bugfix does not need much information and multiple unrelated headings would somewhat hide the important information IMO. Will change to one comment per heading. 

I think the only risk is, that someone external would just ignore all the commented stuff as not mandatory. Since I think it is very clear when to fill out a specific section, we won't run into this issue. 

Furthermore, most repositories like openenclave, golang/go, occlum don't employ a template at all and graphene's is also not that verbose.